packages/server/tests/acceptance/htsPrecompile_v1.spec.ts

```diff
@@ -0,0 +1,149 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// external resources
+import { solidity } from 'ethereum-waffle';
+import chai, { expect } from 'chai';
+
+chai.use(solidity);
+
+import { AliasAccount } from '../clients/servicesClient';
+import { ethers } from 'ethers';
+import ERC20MockJson from '../contracts_v1/ERC20Mock.json';
```

Comment:
Let's keep a single main `contract` folder and have sub folders instead
```suggestion
import ERC20MockJson from '../contracts/htsPrecompileV1/ERC20Mock.json';
```